runtimes/shared-configuration/src/funding.rs

```diff
@@ -86,7 +86,7 @@ pub const SUCCESS_TO_SETTLEMENT_TIME: BlockNumber = 4;
 #[cfg(feature = "fast-mode")]
 pub const SUCCESS_TO_SETTLEMENT_TIME: BlockNumber = 5 * crate::MINUTES;
 #[cfg(not(any(feature = "fast-mode", feature = "instant-mode")))]
-pub const SUCCESS_TO_SETTLEMENT_TIME: BlockNumber = 4 * crate::DAYS;
+pub const SUCCESS_TO_SETTLEMENT_TIME: BlockNumber = 1 * HOURS;
```

Comment:
Are we completely clear about this change? I understood that the 4 days were for some legal reasons, right? Just wanted to double check that this was cleared with Ken.

Reply:
Yes, we are clear. This change follows a discussion we had with the legal team